config/deploy/dev.rb

```diff
@@ -0,0 +1,7 @@
+set :deploy_host, ENV['CAPISTRANO_DEVELOPMENT_DEPLOY_HOST'] || 'library-hours.stanford.edu'
```

Comment:
I wouldn't bother with the `env` vars at all. It was just a way for me to avoid getting prompted every time.